app/packages/annotation/src/components/SaveStatusIndicator.module.css

```
@@ -0,0 +1,35 @@
+/* Slow opacity pulse for the in-flight state. A local keyframe until a shared
+ * animation token lands in voodo. */
+@keyframes saveStatusPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "saveStatusPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)